test/JoeMakerV4.test.ts

```diff
@@ -115,7 +115,7 @@ describe("joeMakerV4", function () {
     it("should convert WAVAX", async function () {
       await this.wavax.deposit({ value: ethers.utils.parseEther("2") })
       await this.wavax.transfer(this.joeMakerV4.address, await this.wavaxERC20.balanceOf(this.dev.address))
-      await this.joeMakerV4.convert(this.wavax.address, this.wavax.address)
+      await this.joeMakerV4.convert(this.wavax.address, this.wavax.address, "100")
       expect(await this.usdc.balanceOf(this.joeMakerV4.address)).to.equal(0)
       expect(await this.wavaxERC20.balanceOf(this.joeMakerV4.address)).to.equal(0)
       expect(await this.usdc.balanceOf(BAR_ADDRESS)).to.equal("215434297")
```

Comment:
I don't quite get what you're testing here

Reply:
JoeMakerV4 uses USDC